src/declarations.d.ts

```diff
@@ -1,3 +1,7 @@
-declare let fathom
-declare let Desmos
-declare module 'ethereum-block-by-date'
+export declare let Desmos
+export declare global {
```

Comment:
https://bobbyhadz.com/blog/typescript-property-does-not-exist-on-type-window#fix---property-does-not-exist-on-type-window-in-typescript the article I found on using third party modules.